### ISSUE TITLE
[release-blocking] anago fails to properly delete old CHANGELOGs on new branch creation

### ISSUE THREAD

**Issue by justaugustus** [kubernetes/release#1100]
**What happened**:
As `anago` now has me prone to default to paranoia, I wanted to run a [mock stage of the branch cut](https://console.cloud.google.com/cloud-build/builds/8092b9b8-3fa6-4991-94b2-c5ce175221b4?project=kubernetes-release-test). 

Sure enough, it revealed that the CHANGELOG location move in https://github.com/kubernetes/kubernetes/pull/87879 causes a failure when trying to remove the previous release changelogs on a new branch:

```shell
Step #2: ================================================================================
Step #2: PREPARE AND TAG TREE beta (5/6)
Step #2: ================================================================================
Step #2: 
Step #2: Checking out release-1.18 (-b release-1.18 c099585b10cb8c): OK
Step #2: Remove any previous CHANGELOG-*.md files: FAILED
Step #2: [2020-Feb-13 08:16:21 UTC] prepare_tree in 0s
Step #2: FAILED in prepare_tree.
Step #2: 
Step #2: RELEASE INCOMPLETE! Exiting...
```

Basically, we search/replaced the changelog in most places, but this code branch has a wildcard in it, which is likely why it was missed.

Fix has been opened here: https://github.com/kubernetes/release/pull/1099

Assigning the Branch Managers:
/assign @cpanato @saschagrunert 
cc: @kubernetes/release-engineering @kubernetes/release-team-leads 
/priority critical-urgent

**Comment by justaugustus** [comment]
@saschagrunert @cpanato -- Once https://github.com/kubernetes/release/pull/1099 merges, I'd like you to sanity check the following permutations, _before closing this_:

- alpha mock stage/release (`master`)
- beta mock stage/release (`release-1.18`)
- rc mock stage/release (`release-1.15`)
- official mock stage/release (`release-1.15`)

It makes sense for each of you to handle two of these scenarios apiece.

**Comment by cpanato** [comment]
@justaugustus maybe we can split, lets sync over chat

**Comment by justaugustus** [comment]
Yep, thread for this issue is here: https://kubernetes.slack.com/archives/C2C40FMNF/p1581672007179100

**Comment by justaugustus** [comment]
Closed via https://github.com/kubernetes/release/pull/1099 and verified by @cpanato & @saschagrunert last week.
/close
